engines/decidim-debates/app/views/decidim/debates/debates/show.html.erb

```diff
@@ -0,0 +1,66 @@
+<% content_for :meta_description, debate.description %>
+<% content_for :meta_title, debate.title %>
+<% content_for :meta_url, debate_url(debate.id) %>
+<% content_for :twitter_handler, current_organization.twitter_handler %>
+
+<div class="wrapper">
+  <div class="row column view-header">
+    <h2 class="heading2">
+      <%== translated_attribute debate.title %>
+    </h2>
+    <div class="author-data">
+      <div class="author-data__main">
+        <div class="author author--inline">
+          <a href="#" class="author__avatar">
+            <img src="<%= debate.author_avatar_url %>" alt="">
+          </a>
+          <%= l debate.created_at, format: :long %>
+          <a href="#" class="author__name">
+            <%= debate.author_name %>
+          </a>
+        </div>
+      </div>
+      <div class="author-data__extra">
+        <a href="#comments" title="<%= t('.comments') %>">
+          <%= icon "comment-square", aria_label: t('.comments'), role: "img" %> <%= Decidim::Comments::Comment.where(commentable: debate).count %>
```

Comment:
I'd add a relation at debate, something like:

`has_many :comments, as: :commentable`

So you can just do `debate.comments.count`

/cc @josepjaume @beagleknight